UniMath/CategoryTheory/.package/files

```diff
@@ -314,3 +314,6 @@ Monoidal/RezkCompletion/LiftedUnitors.v
 Monoidal/RezkCompletion/LiftedAssociator.v
 Monoidal/RezkCompletion/LiftedMonoidal.v
 Monoidal/RezkCompletion/MonoidalRezkCompletion.v
+
+PowerObject.v
+elementary_topos.v
```

Comment:
I believe it is now more common to use CamelCase file names rather than names with underscores. So maybe rename `elementary_topos.v` into `ElementaryTopos.v`?

Reply:
Ok, just committed the renaming. Will keep that in mind for the future.